WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/ProductDetailCardBuilder.kt

```diff
@@ -459,8 +459,12 @@ class ProductDetailCardBuilder(
     private fun Product.productTypeDisplayName(): String {
         return when (productType) {
             SIMPLE -> {
-                if (this.isVirtual) resources.getString(R.string.product_type_virtual)
-                else resources.getString(R.string.product_type_physical)
+                if (this.isDownloadable) {
+                    resources.getString(R.string.product_type_downloadable_label)
+                } else {
+                    if (this.isVirtual) resources.getString(R.string.product_type_virtual)
+                    else resources.getString(R.string.product_type_physical)
+                }
```

Comment:
np: we can simplify this as the following:
```suggestion
                when {
                    this.isDownloadable -> resources.getString(R.string.product_type_downloadable_label)
                    this.isVirtual -> resources.getString(R.string.product_type_virtual)
                    else -> resources.getString(R.string.product_type_physical)
                }
```